internal/component/otelcol/receiver/prometheus/internal/prom_to_otlp.go

```diff
@@ -10,12 +10,15 @@ import (
 	"github.com/prometheus/prometheus/model/labels"
 	"go.opentelemetry.io/collector/featuregate"
 	"go.opentelemetry.io/collector/pdata/pcommon"
-	conventions "go.opentelemetry.io/collector/semconv/v1.25.0"
-	oldconventions "go.opentelemetry.io/collector/semconv/v1.6.1"
+	conventions "go.opentelemetry.io/otel/semconv/v1.25.0"
+	oldconventions "go.opentelemetry.io/otel/semconv/v1.6.1"
 )
 
 const removeOldSemconvFeatureGateID = "receiver.prometheusreceiver.RemoveLegacyResourceAttributes"
 
+// TODO: Find a way to enable this feature gate again, at least for 1-2 more versions of Alloy.
+// Those conventions were deprecated a really long time ago:
+// https://www.honeycomb.io/blog/opentelemetry-http-attributes
```

Comment:
Just out of curiosity, what is stopping us from removing this now? Would we break a lot of things?